ticket-proposal/tickets_idea_and_questions.md

```diff
@@ -0,0 +1,88 @@
+# Tickets Ideas and Questions
+
+## Design Considerations
+
+### Ticket Events
+Do we need an actual ticket created event?
```

Comment:
Would the config change not go through source control?
How would this event be created in that case?

Reply:
> To answer this question it might be helpful to talk about how we want to handle approvals. 

Yea, I was thinking the same thing.

We talked about use cases last time, but I think it may be a good idea to expand on the overall flow of how we envision tickets being handled through CDEvents and the ultimate goal.

> 1. Send approval as a ticket updated event.
For example is this an important goal for tickets in CDEvents? So lots to talk about this upcoming meeting :)